microgen/remesh.py

```diff
@@ -0,0 +1,166 @@
+import gmsh
+from microgen import Rve, Mmg
+import numpy as np
+import math as m
+from collections import namedtuple
+from tempfile import NamedTemporaryFile
+
+Triangle = namedtuple("Triangle", ["node1", "node2", "node3", "tag"])

```

Comment:
`namedtuple`s are great! Why not using the `typing` syntax though?

```python
from typing import NamedTuple
import numpy.typing as npt


class Triangle(NamedTuple):
    node1: npt.NDArray[np.float_]
    node2: npt.NDArray[np.float_]
    node3: npt.NDArray[np.float_]
    tag: int
```

It allows to add type annotations for each of the tuple elements, and avoids to repeat `Triangle` twice